oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -3,9 +3,9 @@ OEP-19: Developer Documentation
 #################################
 
 .. list-table::
-  
+
    * - OEP
-     - :doc:`OEP-0019 </oeps/oep-0019>`
+     - :doc:`OEP-0019 </oeps/oep-0019-bp-developer-documentation>`
```

Comment:
**Reviewer Note**: This was the only real change.  The rest were just from my editor removing spaces.